pints/_log_pdfs.py

```diff
@@ -50,12 +51,15 @@ def __call__(self, x):
     def sample(self, n=1):
         """
         Returns ``n`` random samples from the underlying prior distribution.
+
+        The returned value is a numpy array with shape ``(n, d)`` where ``n``
```

Comment:
So even `n=1` it would be in the shape of (1, d)? And if `d=1`, we are not returning a scalar float number but an array of 1 value?

Reply:
Yup! This way it's always consistent!